org.eclipse.corrosion/src/org/eclipse/corrosion/wizards/newproject/NewCargoProjectWizard.java

```diff
@@ -205,6 +216,10 @@ public boolean performFinish() {
 		return true;
 	}
 
+	private void switchPerspective() {
```

Comment:
I don't think we need a method for a 1-line instruction. Let's just put this instruction directly in the code.